app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -238,15 +244,21 @@ def image_inspector_client
   end
 
   def queue_signal(*args)
-    MiqQueue.put_unless_exists(
-      :args        => args,
-      :class_name  => "Job",
+    cb = {
+      :class_name  => self.class.to_s,
       :instance_id => id,
-      :method_name => "signal",
-      :priority    => MiqQueue::HIGH_PRIORITY,
-      :role        => "smartstate",
-      :task_id     => guid,
-      :zone        => zone
+      :method_name => :queue_callback
+    }
+    MiqQueue.put(
```

Comment:
Some strange bug from ActiveRecord is preventing me from using it:
https://github.com/ManageIQ/manageiq/pull/9040#issuecomment-222648892